File: LyricsStudio/DebugWindow.cs
using System;
using Microsoft.VisualBasic;

namespace ti_Lyricstudio
{
    public partial class DebugWindow
    {
        public DebugWindow()
        {
            InitializeComponent();
        }
        public void AddDLine(string Text, int Grade = 0)
        {
            // Grade    0 = Information, 1 = Warning, 2 = Error
            if (Grade == 2)
            {
                RichDebug.Text += "E ";
            }
            else if (Grade == 1)
            {
                RichDebug.Text += "W ";
            }
            else
            {
                RichDebug.Text += "I ";
            }
            RichDebug.Text += Text + Constants.vbNewLine;
        }

        private void DebugWindow_Resize(object sender, EventArgs e)
        {
            RichDebug.Width = Width;
            RichDebug.Height = Height;
        }
    }
}

File: LyricsStudio/DebugWindow.vb
Public Class DebugWindow
    Public Sub AddDLine(ByVal Text As String, Optional ByVal Grade As Integer = 0)
        'Grade    0 = Information, 1 = Warning, 2 = Error
        If Grade = 2 Then
            RichDebug.Text += "E "
        ElseIf Grade = 1 Then
            RichDebug.Text += "W "
        Else
            RichDebug.Text += "I "
        End If
        RichDebug.Text += Text + vbNewLine
    End Sub

    Private Sub DebugWindow_Resize(sender As Object, e As EventArgs) Handles Me.Resize
        RichDebug.Width = Width
        RichDebug.Height = Height
    End Sub
End Class